livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -105,8 +90,21 @@ class RealtimeToolCall:
     """id of the tool call"""
 
 
-# TODO(theomonnom): add the content type directly inside RealtimeContent?
-# text/audio/transcript?
+@dataclass
+class Capabilities:
+    supports_chat_ctx_manipulation: bool
```

Comment:
looking at what this is used for, it seems like it's only used for messages truncation? should this just be: `supports_truncate`?

Reply:
@jayeshp19